cmake/modules/RootMacros.cmake

```diff
@@ -609,7 +609,11 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
 
     # get target properties added after call to ROOT_GENERATE_DICTIONARY()
     if(TARGET ${ARG_MODULE})
-      set(module_incs $<TARGET_PROPERTY:${ARG_MODULE},INCLUDE_DIRECTORIES>)
+      if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.15)
+        set(module_incs $<REMOVE_DUPLICATES:$<TARGET_PROPERTY:${ARG_MODULE},INCLUDE_DIRECTORIES>>)
+      else()
```

Comment:
Can we 'assert' if the minimal version on unix goes  `VERSION_GREATER_EQUAL 3.15` to drop that branch?

Reply:
I think when we change the minimal required version, one should grep for this kind of check and remove them...